Configuration/Metadata/Driver/AnnotationDriver.php

```diff
@@ -13,71 +13,254 @@
 
 use Doctrine\Common\Annotations\Reader;
 use Doctrine\Common\Util\Inflector;
+use JMS\Serializer\Annotation\Accessor;
+use JMS\Serializer\Annotation\AccessorOrder;
+use JMS\Serializer\Annotation\AccessType;
+use JMS\Serializer\Annotation\Discriminator;
+use JMS\Serializer\Annotation\Exclude;
+use JMS\Serializer\Annotation\ExclusionPolicy;
+use JMS\Serializer\Annotation\Expose;
+use JMS\Serializer\Annotation\Groups;
+use JMS\Serializer\Annotation\HandlerCallback;
+use JMS\Serializer\Annotation\Inline;
+use JMS\Serializer\Annotation\MaxDepth;
+use JMS\Serializer\Annotation\PostDeserialize;
+use JMS\Serializer\Annotation\PostSerialize;
+use JMS\Serializer\Annotation\PreSerialize;
+use JMS\Serializer\Annotation\ReadOnly;
+use JMS\Serializer\Annotation\SerializedName;
+use JMS\Serializer\Annotation\Since;
+use JMS\Serializer\Annotation\Type;
+use JMS\Serializer\Annotation\Until;
+use JMS\Serializer\Annotation\VirtualProperty;
+use JMS\Serializer\Annotation\XmlAttribute;
+use JMS\Serializer\Annotation\XmlAttributeMap;
+use JMS\Serializer\Annotation\XmlDiscriminator;
+use JMS\Serializer\Annotation\XmlElement;
+use JMS\Serializer\Annotation\XmlKeyValuePairs;
+use JMS\Serializer\Annotation\XmlList;
+use JMS\Serializer\Annotation\XmlMap;
+use JMS\Serializer\Annotation\XmlNamespace;
+use JMS\Serializer\Annotation\XmlRoot;
+use JMS\Serializer\Annotation\XmlValue;
+use JMS\Serializer\Exception\InvalidArgumentException;
+use JMS\Serializer\GraphNavigator;
+use JMS\Serializer\Metadata\ExpressionPropertyMetadata;
+use JMS\Serializer\Metadata\PropertyMetadata;
+use JMS\Serializer\Metadata\VirtualPropertyMetadata;
 use Mango\Bundle\JsonApiBundle\Configuration\Annotation;
 use Mango\Bundle\JsonApiBundle\Configuration\Metadata\ClassMetadata;
 use Mango\Bundle\JsonApiBundle\Configuration\Relationship;
 use Mango\Bundle\JsonApiBundle\Configuration\Resource;
 use Mango\Bundle\JsonApiBundle\Util\StringUtil;
 use Metadata\Driver\DriverInterface;
+use Metadata\MethodMetadata;
 
 /**
  * @author Steffen Brem <steffenbrem@gmail.com>
  */
 class AnnotationDriver implements DriverInterface
```

Comment:
I might re-think this, to avoid copying most of the content from the JMS. Suggestions are appreciated :)